params/config.go

```diff
@@ -486,6 +486,41 @@ func (c *ChainConfig) String() string {
 	return banner
 }
 
+// Summary returns a single-line summary of ChainConfig.
+func (c *ChainConfig) Summary() string {
+	var engine interface{}
+	switch {
+	case c.Ethash != nil:
+		engine = c.Ethash
+	case c.Clique != nil:
+		engine = c.Clique
+	default:
+		engine = "unknown"
+	}
+	return fmt.Sprintf("{ChainID: %v, Homestead: %v, DAO: %v, DAOSupport: %v, EIP150: %v, EIP155: %v, EIP158: %v, Byzantium: %v, Constantinople: %v, Petersburg: %v, Istanbul: %v, Muir Glacier: %v, Berlin: %v, London: %v, Arrow Glacier: %v, MergeFork: %v, ShanghaiFork: %v, CancunFork: %v, Terminal TD: %v, Engine: %v}",
```

Comment:
I think this is kind of equivalent to just `fmt.Sprintf("%+v", c)`

Reply:
You're right. I've fixed it.